### PR TITLE
Replace old Travis CI config with Github Actions

### PR DESCRIPTION
**Pull request by lavaturtle**
What it says on the tin.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# action_kit_rest
+# action\_kit\_rest
```
Comment (anero) [inline README.md:1]: Why is this needed?

**Review by anero — APPROVED** [review]
Could you also update the ruby version while you're at this?

File: README.md (hunk 1 above)
Comment (lavaturtle) [reply README.md:1]: This tells Markdown that these are real underscores, not an attempt to underline/italicize something. It's probably not strictly necessary, but my editor was highlighting it in red and this made it stop :slightly_smiling_face:
